src/components/navigation/mobile/MenuLeft.tsx

```diff
@@ -138,7 +138,7 @@ export function MenuLeft(props: Props) {
 
   async function clearAnalytics(event: UIEvent) {
     recordClick(event);
-    await flushEvents();
+    await resetAnalytics();
```

Comment:
resets anonymousID.

This does not flush the events to rudderstack. Rudderstack automatically flushes events on a set interval